tests/test_examples.py

```diff
@@ -24,57 +24,82 @@
 
 import os
 import matplotlib
-matplotlib.use('Agg')
-
 import matplotlib.pyplot as plt
-
 import dense as ds
+matplotlib.use('Agg')
 
 folder = os.path.dirname(__file__)
 folder = folder if folder else "."
 
-root   = os.path.abspath(folder + "/..")
-tuto   = root + "/examples/tutorials"
+root = os.path.abspath(folder + "/..")
+neuron_models = root + "/examples/models/neurons"
 
 
 def mock_show():
     pass
 
 
-def test_1_first_steps(monkeypatch):
+def test_1_bipolar_cell(monkeypatch):
+    '''
+    Run first example.
+    '''
+    monkeypatch.setattr(plt, "show", mock_show)
+    ds.reset_kernel()
+    exec(open(models + "/bipolar_cell.py").read())
```

Comment:
this should probably be ``neuron_models``, see also below

Reply:
indeed